build_debian.sh

```diff
@@ -445,6 +445,10 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'docker
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install gcc libpython2.7-dev
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'netifaces==0.10.7'
 
+# Get package to support Dynamic Port Breakout
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install xmltodict==0.12.0
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install jsondiff==1.2.0
```

Comment:
this does not seem to be related to vs docker.

Reply:
Yeah It is not, let me start another PR for it, since these 2 PKGs are needed on Sonic Switch to run Sonic_utilities/config/config_mgmt.py Library. Thx

Reply:
Fixed.